contracts/modules/token/TokenController.sol

```diff
@@ -137,17 +137,16 @@ contract TokenController is Iupgradable {
   /**
   * @dev Locks a specified amount of tokens,
   *    for CLA reason and for a specified time
-  * @param _reason The reason to lock tokens, currently restricted to CLA
   * @param _amount Number of tokens to be locked
   * @param _time Lock time in seconds
   */
-  function lock(bytes32 _reason, uint256 _amount, uint256 _time) public checkPause returns (bool)
+  function lockClaimAssessmentTokens(uint256 _amount, uint256 _time) external checkPause returns (bool)
```

Comment:
Since this is a new function that doesn't have to implement the interface, let's get rid of return